phantomgames/Search/SearchGameRepository.swift

```diff
@@ -9,11 +9,16 @@ import Foundation
 typealias SearchGameResult = (Result<[Game], APIError>) -> Void
 
 protocol SearchGameRepositoryType: AnyObject {
+    func fetchAPIImage(completion: @escaping(SearchGameResult))
     func fetchSearchResults(completion: @escaping(SearchGameResult))
 }
 
 class SearchGameRepository: SearchGameRepositoryType {
-    
+    func fetchAPIImage(completion: @escaping (SearchGameResult)) {
+        URLSession.shared.fetchingAPIImages(URL: Constants.Endpoints.search) {
+            games in completion(.success(games))
```

Comment:
is there a swift lint warning message here about this? :)
This should typically look like

URLSession.shared.fetchingAPIImages(URL: Constants.Endpoints.search) { games in 
completion(.success(games))